pkgs/top-level/release.nix

```diff
@@ -55,6 +56,7 @@
 let
   release-lib = import ./release-lib.nix {
     inherit supportedSystems scrubJobs nixpkgsArgs;
+    packageSet = nixpkgs.legacyPackages.${system};
```

Comment:
This intermediate commit is not correct because of this (which you're undoing again in the next commit).

Reply:
Yeah, I'm trying to remove that but git problems.

Reply:
Fixed